src/components/DataGrid/DataGrid.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {
+  DataGrid as MuiDataGrid,
+  DataGridProps,
+  GridColDef,
+  GridRowsProp,
+} from '@mui/x-data-grid';
+
+import theme from '../../theme';
+import { makeStyles } from '../../theme';
+import Box from '../Box';
+
+export interface Props extends DataGridProps {
+  columns: GridColDef[];
+  rows: GridRowsProp;
+  pageSize?: number;
+  height: string;
+}
+
+const useStyles = makeStyles({
```

Comment:
should be able to use `styled` here; don't want to use `makeStyles` anymore!

the css classes should be able to be used the same way